### PR TITLE
[oneDNN][Bug-fix] Add check for min-max tensors in quantized matmul OpKernel.

### PR DESCRIPTION
**Pull request by mdfaijul**
This PR fixes NPE (null-pointer exception error) on tensors relevant to quantized matmul OpKernel. Particularly, when an element is accessed with an index, this PR first checks if such index is valid or not.

### CODE REVIEW

**Comment by mdfaijul** [comment]
@penpornk This PR has a fix on `tf.raw_ops.QuantizedMatMulWithBiasAndDequantize`. Please help reviewing this. Thanks!

File: tensorflow/core/kernels/mkl/mkl_qmatmul_op.cc [hunk 1]
```diff
@@ -361,6 +373,25 @@ class MklDnnQuantizedMatMulOp : public MklDnnMatMulOpBase<Tweight, Toutput> {
...
+    OP_REQUIRES(context, TensorShapeUtils::IsScalar(max_input_tensor.shape()),
+                errors::InvalidArgument("`max_a` must be rank 0 but is rank ",
+                                        max_input_tensor.dims()));
+    OP_REQUIRES(context, TensorShapeUtils::IsScalar(min_weight_tensor.shape()),
+                errors::InvalidArgument("`min_b` must be rank 0 but is rank ",
+                                        min_weight_tensor.dims()));
+    OP_REQUIRES(context, TensorShapeUtils::IsScalar(max_weight_tensor.shape()),
+                errors::InvalidArgument("`max_b` must be rank 0 but is rank ",
+                                        max_weight_tensor.dims()));
+
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl/mkl_qmatmul_op.cc:394]: Shouldn't this check be before lines 359-362?

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you very much for the quick fix!

File: tensorflow/core/kernels/mkl/mkl_qmatmul_op.cc (hunk 1 above)
Comment (penpornk) [reply tensorflow/core/kernels/mkl/mkl_qmatmul_op.cc:394]: Oops. Please ignore my comment. I misread and thought they were in the same function.

**Review by penpornk — APPROVED** [review]

**Comment by penpornk** [comment]
[MacOS CPU Python3](https://source.cloud.google.com/results/invocations/709f8779-1cf0-4c10-a8cd-fc06ffcddd52/targets) failures seem unrelated to this PR:
```
clang++: error: no such file or directory: 'tensorflow/compiler/mlir/lite/sparsity/sparsify_model.cc'
clang++: error: no input files
```

**Pull request merged**